lib/internal/fs/recursive_watch.js

```diff
@@ -202,6 +205,10 @@ class FSWatcher extends EventEmitter {
         this.emit('change', 'rename', pathRelative(this.#rootPath, file));
       } else if (currentStats.isDirectory()) {
         this.#watchFolder(file);
+      } else {
+        // Watching a directory will trigger a change event for child files)
+        const event = DateNow() - currentStats.birthtimeMs < NEW_FILES_AGE ? 'rename' : 'change';
```

Comment:
Isn't this boolean expression better for detecting the creation of files? 

`currentStats.birthTimeMs !== 0 && previousStats.birthtimeMs === 0`